### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by stephan281094**
- Removed Global CSS (screen-specific CSS is still included).
- Removed state management setup.
- Moved components folders into files.
- Changed dev port to 3000.

I'm aware that this breaks many things, but at least now global styles are no longer interfering with screens. I added a bunch of FIXMEs in the code, which can be picked up independently by people. In the meantime you can continue working on the different screens.

Tomorrow I plan on added the initial part of the animal stats screen, and will then work on FIXMEs like the navigation component and others.

### CODE REVIEW

File: src/components/dialog.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import * as RadixDialog from "@radix-ui/react-dialog";
+import { forwardRef } from "react";
+
+// FIXME
```
Comment (trybick) [inline src/components/dialog.js]: Would it make sense to add some context on what the issue is/what should be fixed?

File: src/components/github-icon.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+export function GithubIcon({ size = 48, ...props }) {
+  return (
+    <svg
```
Comment (trybick) [inline src/components/github-icon.js:3]: Great idea using react components for svg!

File: src/components/navigation.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+export function Navigation() {
+  const [open, setOpen] = useState(false);
+
+  // Close the menu when clicking on a link.
+  useEffect(() => {
+    const onHashChange = () => {
+      setOpen(false);
+    };
+
+    window.addEventListener("hashchange", onHashChange);
```
Comment (trybick) [inline src/components/navigation.js:19]: Do we also want to close the nav menu if the user clicks outside of it (maybe that for later)?

File: src/components/dialog.js (hunk 1 above)
Comment (stephan281094) [reply src/components/dialog.js]: Yea, good call. I realize this is not very descriptive for anyone 😬

File: src/components/navigation.js (hunk 3 above)
Comment (stephan281094) [reply src/components/navigation.js:19]: That's a great idea, yes! I'm actually thinking about reimplementing this component with something from Radix UI, where these kind of things are automatically handled for us. Another thing, for instance, is closing the menu when the user pressed Escape.

**Review by trybick — APPROVED** [review]

**Pull request merged**